autoload/hacx.wad/bloodcolor.deh

```diff
@@ -0,0 +1,8 @@
+Thing 16 (Baron of Hell)
```

Comment:
How about the Cacodemon?

Reply:
Hacx doesn't have Cacodemon. Thing 15 is some decoration here.

Reply:
Ok, thanks, didn't even know!